src/mach/templates/partials/commercetools.tf

```diff
@@ -18,6 +18,7 @@ resource "commercetools_project_settings" "project" {
     messages {
         enabled = {{ commercetools.project_settings.messages_enabled | string | lower }}
     }
+    enable_search_index_products = true
```

Comment:
This is too opinionated.
If you don't want to enable this feature, there is no way of turning it of, so actually might have an unwanted side-effect.

Ideally we make this another project setting within MACH, and make sure the default matches the commercetools provider default.

Reply:
I've added a settings called site_search_index_enabled so that this can be configured using the MACH config